src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/ExternalServiceEntity.kt

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.FetchType
+import jakarta.persistence.Id
+import jakarta.persistence.OneToMany
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+
+@Entity
+@Table(name = "EXTERNAL_SERVICES")
+data class ExternalServiceEntity(
```

Comment:
We have a theory that AuditableEntity is not needed as all created and modified columns are populated by Nomis - will test this on t3.